tests/test_utils.py

```diff
@@ -1,12 +1,6 @@
 import sys
 
-if sys.version_info >= (3, 10):
```

Comment:
don't remove that. It's part of the test for the production code: https://github.com/fake-useragent/fake-useragent/blob/1b8c2ccfcf8c208a73c1a96c740b9443708775f9/src/fake_useragent/utils.py#L8

Reply:
Restored the older test, but to have it functional, setuptools cannot exceed a specific version 70.3.0. Better description is provided below